src/containers/ClassifierContainer.jsx

```diff
@@ -87,7 +88,9 @@ class ClassifierContainer extends React.Component {
             <button href="#" className="white-red button">Your Crib Sheet</button>
             <img className="divider" role="presentation" src={Divider} />
             <button href="#" className="white-green button" onClick={this.completeClassification}>Done</button>
-            <button href="#" className="green button" onClick={this.completeClassification}>Done &amp; Talk</button>
+            <button href="#" className="green button" onClick={this.completeClassification}>
+              <a href={config.zooniverseLinks.host + 'projects/' + config.zooniverseLinks.projectSlug + '/talk'} target="_blank">Done &amp; Talk</a>
```

Comment:
Does clicking on this link properly open the Talk window for you? I can't seem to get this working on OSX+Chrome; I suspect the parent <button> is consuming the event before it even hits the <a>

Reply:
Either that or my popup blocker is quite voracious.